Library/Homebrew/cmd/test-bot.rb

```diff
@@ -590,6 +590,11 @@ def formula(formula_name)
           if bottle_step.passed? && bottle_step.has_output?
             bottle_filename =
               bottle_step.output.gsub(/.*(\.\/\S+#{bottle_native_regex}).*/m, '\1')
+            bottle_rb_filename = bottle_filename.gsub(/\.(\d+\.)?tar\.gz$/, ".rb")
+            bottle_merge_args = ["--merge", "--write", "--no-commit", bottle_rb_filename]
```

Comment:
But we can make it use it if we want :wink: 

Therefore, it may help to speed up process when building multiple formulae depend on each other. For example, gcc.